ui/src/components/App.jsx

```diff
@@ -68,10 +69,11 @@ const router = createBrowserRouter([
 
 function App() {
   const dispatch = useDispatch();
+  const { showBoundary } = useErrorBoundary();
   const loggedIn = useSelector((state) => state.login.loggedIn);
 
   useEffect(() => {
-    dispatch(getLoginInfo());
+    dispatch(getLoginInfo()).catch(showBoundary); // eslint-disable-line promise/prefer-await-to-then
```

Comment:
Obviously there are a lot more cases of uncaught async errors in the application, but at least now we have a way to deal with them if need be.

> The long term goal is to not use Redux for data fetching and take advantage of React Suspense instead (so errors are thrown during render and caught by error boundaries).